modules/ROOT/pages/proxy-latest-concept.adoc

```diff
@@ -1,30 +1,66 @@
 
-You can import a proxy that you download from API Manager into Studio as a Mule Deployable Archive. You can then modify the proxy in Studio.
+//Kim, IMO, not mentioning that no prerequisites are required for CH might be confusing for the users, so I addedd it. What do you think?
 
-If you configured an endpoint with a proxy based on Mule 3 to use the HTTPS scheme, you need to modify a proxy to add HTTPS credentials following the xref:mule-runtime::build-an-https-service.adoc[Build an HTTPS Service] instructions. +
-Additionally, if you configured a proxy based on Mule 4, you can build an HTTPS proxy using a TLS context stored in Anypoint Security's secrets manager following the xref:building-https-proxy.adoc[Building an HTTPS API Proxy] instructions.
+== Deploy your API Proxy
 
+To deploy your proxy:
 
+. From Anypoint Platform, select *API Manager* > *Manage API* > *Manage API From Exchange*.
+. In *API Name*, search for the name of the asset as saved in Exchange.
++
+The *Asset type*, *API version*, and *Asset version* fields for the API are auto-populated from Exchange.
+. From *Managing type*, select *Endpoint with proxy* and select the *Proxy deployment target* for the proxy, *CloudHub*, *Hybrid*, or *Runtime Fabric*:
++
+image:select-deployment-target.png[Select Managing type and Deployment Target]
 
-Now, you're ready to deploy an auto-generated proxy.
+<1> Search for APIs from Exchange.
+<2> Select a proxy deployment target.
+. If your Mule target is Mule 4, check *Mule version* or leave it unchecked if you are using Mule 3.
++
+If you are deploying your API to a Mule 4 environment and want to enable the proxy to support HTTPS requests, you must first xref:building-https-proxy.adoc[build HTTPS API proxies]. 
+. In *Implementation URI*, specify the base URL path using which you can access the proxy.
+. Optionally, specify the advanced parameters, such as *TLS context for outbound traffic*, *Proxy version*, *Validations*, *Response timeout*, and *Port*.
+. Click *Save*.
++
+Your configuration information is saved and the *Deployment Configuration* section of the UI is displayed.
+. In *Deployment Configuration*, specify the Mule version in *Runtime version* and the name of the proxy application in *Proxy application name*.
++
+If you are updating an existing proxy application, select *Update application if it exists*.
+. Click *Deploy*.
++
+Your API proxy is now deployed to your selected environment. The following illustration shows an example of the screen that appears when you deploy an API proxy to CloudHub:
++
+image:deploy-api-proxy.png.png[Example of Deploying API Proxies in CloudHub,80%,80%]
+
+<1> Deploying the API to a specific enviornment, for example, CloudHub in this case.
+<2> Deployment status displayed as successful.
+<3> View deployment logs in Runtime Manager.
+. If you deployed your API to a Mule 3 environment, xref:enable-https-mule3-proxies.adoc[enable HTTPS in Mule 3x].
```

Comment:
```suggestion
. If you deployed your API to a Mule 3 environment, xref:enable-https-mule3-proxies.adoc[enable HTTPS].
```

Reply:
Related to Line 44? Should that info be here instead?

Reply:
Done! Thanks!